### PR TITLE
Ensure bindings for computeSubModelMatrixRelatingFTSensorsMeasuresAndKinematics are working fine

### PR DESCRIPTION
**Pull request by traversaro**
Fix https://github.com/robotology/idyntree/issues/1102 . The `bindings/python/tests/ExtWrenchesAndJointTorquesEstimatorUnitTest.py` show an example of how to use this function, that can be easily converted to MATLAB if necessary. After merging this PR, I will regenerate the bindings.

### CODE REVIEW

File: bindings/python/tests/ExtWrenchesAndJointTorquesEstimatorUnitTest.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        consideredJoints.push_back("r_hip_pitch")
+        consideredJoints.push_back("r_hip_roll")
+        consideredJoints.push_back("r_hip_yaw")
+        consideredJoints.push_back("r_knee")
+        consideredJoints.push_back("r_ankle_pitch")
+        consideredJoints.push_back("r_ankle_roll")
+
+        # Load model
+        estimator = iDynTree.ExtWrenchesAndJointTorquesEstimator()
+        testModel = os.path.join(os.path.dirname(__file__), "../../../../src/tests/data/iCubDarmstadt01.urdf")
```
Comment (HosameldinMohamed) [inline bindings/python/tests/ExtWrenchesAndJointTorquesEstimatorUnitTest.py:62]: Is the test supposed to run from its current directory?

In this case the model file should be:

```
testModel = os.path.join(os.path.dirname(__file__), "../../../src/tests/data/iCubDarmstadt01.urdf")
```

Comment (traversaro) [reply bindings/python/tests/ExtWrenchesAndJointTorquesEstimatorUnitTest.py:62]: Sorry, I thought I replied. No, that file is copied in `build/bindings/python/tests`  to run. Indeed, that is confusing, I opened https://github.com/robotology/idyntree/issues/1113 to solve this.

**Comment by traversaro** [comment]
fyi @HosameldinMohamed

**Review by HosameldinMohamed — APPROVED** [review]
Thanks!

**Pull request merged**